libbeat/kibana/index.go

```diff
@@ -0,0 +1,121 @@
+package kibana
+
+import (
+	"encoding/json"
+	"errors"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+
+	"github.com/elastic/beats/libbeat/common"
+)
+
+type Index struct {
+	Version   string
+	IndexName string
+	BeatDir   string
+	BeatName  string
+
+	fieldsYaml       string
+	targetDirDefault string
+	targetDir5x      string
+	targetFilename   string
+}
+
+// Create the Index-Pattern for Kibana for 5.x and default.
+func (i *Index) Create() ([]string, error) {
+	indices := []string{}
```

Comment:
this could be moved to line 57 so no allocation happens in case of errors or just have `var indicies []string` here.